spec/mailers/draft_mailer_spec.rb

```diff
@@ -64,6 +66,60 @@
         expect(mail.text_part.body).to have_content("#{user[:name]}, Your collection metadata record #{short_name}_#{version} has been successfully updated in the CMR (test).", normalize_ws: true)
       end
 
+      it 'does not include in the record updated notice published with issues information' do
+        expect(mail.html_part.body).to have_no_content('Your collection was published with the following issues:')
+        expect(mail.text_part.body).to have_no_content('Your collection was published with the following issues:')
```

Comment:
dupe

Reply:
fyi, not a dupe actually. one is checking the html version, one the text version